### PR TITLE
docs/release-process.md: change placeholder value syntax

### PR DESCRIPTION
**Pull request by gotmax23**
Using environment variable syntax makes values easier to copy and paste.

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ This only needs to be done once.
    This guide uses your Github username as the fork remote name.
 
    ```
-   git remote add {USERNAME} https://github.com/{USERNAME}/ansible-build-data
-   git fetch {USERNAME} -v
+   git remote add ${USERNAME} https://github.com/${USERNAME}/ansible-build-data
+   git fetch ${USERNAME} -v
    ```
 
 ## Preform release process
```
Comment (mariolenz) [inline docs/release-process.md:59]:
```suggestion
## Perform release process
```

While your at it, maybe you could also fix this typo :-)

At least, I think that it's a typo and "perform" is meant here.

Comment (gotmax23) [reply docs/release-process.md:59]: Indeed!

File: docs/release-process.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Make sure to mount your working directory as a volume so you don't have to set
 up new repository clones every time.
 
 ```
-podman run --name ansible-release -v {PERSISTENT DIRECTORY}:/pwd:z -w /pwd -ti docker.io/library/python:3.10 bash
+podman run --name ansible-release -v ${PERSISTENT DIRECTORY}:/pwd:z -w /pwd -ti docker.io/library/python:3.10 bash
```
Comment (mariolenz) [inline docs/release-process.md:22]:
```suggestion
podman run --name ansible-release -v ${PERSISTENT_DIRECTORY}:/pwd:z -w /pwd -ti docker.io/library/python:3.10 bash
```

Missing underscore here, I think.

**Review by felixfontein — APPROVED** [review]

**Review by mariolenz — APPROVED** [review]
LGTM

**Comment by gotmax23** [comment]
Thanks for the reviews, @mariolenz and @felixfontein

**Pull request merged**